src/main/java/org/broadinstitute/hellbender/engine/FeatureCache.java

```diff
@@ -223,11 +223,23 @@ public void trimToNewStartPosition( final int newStart ) {
     }
 
     /**
-     * Print statistics about the cache hit rate for debugging
+     * Print statistics about the cache hit rate for debugging.
      */
     public void printCacheStatistics() {
+        printCacheStatistics("");
+    }
+
+    /**
+     * Print statistics about the cache hit rate for debugging.
+     * @param sourceName The source for the features in this cache.
+     */
+    public void printCacheStatistics(final String sourceName) {
```

Comment:
I like the sourceName idea - would support making this a constructor arg since this could for error messages, etc. Alternatively its something I can do in my caching PR it not here.

Reply:
@cmnbroad Sounds good.  I'd like to get this one in sooner rather than later - would you mind putting it in your branch?

Reply:
@jonn-smith Sure, I can do it in my branch.